sidecar/nginx/conf/amalgam8-services.conf.tmpl

```diff
@@ -0,0 +1,33 @@
+include /etc/nginx/amalgam8-access-logging.conf;
+include /etc/nginx/amalgam8-dynupstreams.conf;
+include /etc/nginx/nginx-customizations.conf;
+
+### A8 Sidecar proxy configuration
+server {
+       listen 6379{{.SSLFlag}};
+
+       ## Comment the previous line and enable the latter two if you are using the sidecar as a gateway
+       #listen 80;
+       #listen 443 ssl;
+
+       ## For using the A8 proxy as a gateway with SSL support
+       # eventhough the key and cert are stored in same file, only cert is
+       # sent to client
+       {{.CertPath}}
+       {{.CertKeyPath}}
+
+       location ~ ^/(?<service_name>.*?)(?<reqpath>/(.*))?$ {
+         include /etc/nginx/amalgam8-rules.conf;
+         # Add proxy SSL verification directives here:
```

Comment:
Change this comment to 
```nginx
####DO NOT CHANGE#####
{{.CACertPath}}
######################
```

Apply the same style to other parts of the config that have the template code.